File: RentManagement.Api/Models/Tenant.cs
﻿namespace RentManagement.Api.Models
{
    public class Tenant
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public IEnumerable<RentalAgreement> RentalAgreements { get; set; } = null!;
        public string CreatedByUserId { get; set; } = null!;
    }
}

File: RentManagement.Client/src/app/_models/tenant.ts
import { AgreementWithShop } from './agreement';

export interface TenantDetails {
  id: number;
  name: string;
  phone: string;
  email: string;
}

export interface Tenant {
  id: number;
  name: string;
  phone: string;
  email: string;
  agreements: AgreementWithShop[];
}
